RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -87,7 +87,7 @@
     minPt = 0.2,
 )
 highPtTripletStepTrajectoryFilterBase = _highPtTripletStepTrajectoryFilterBase.clone(
-    maxCCCLostHits = 2,
+    maxCCCLostHits = 0,
     minGoodStripCharge = dict(refToPSet_ = 'SiStripClusterChargeCutLoose')
```

Comment:
Thanks for the confirmation.

given the history of highPtTripletStep (it didn't really exist until after the CCC settings became defaults in 8X, and it only affects phase1 and later),
I thought it was meaningful to put in a separate PR.